tests/improver-ensemble-calibration/09-current_forecast_rebadged_percentiles.bats

```diff
@@ -49,6 +49,6 @@
   # is percentiles that have been rebadged as realizations. The known good
   # output in this case is the same as when passing in percentiles directly
   # and check it passes.
-  run nccmp -dmNsg -x realization "$TEST_DIR/output.nc" \
+  run nccmp -dmNsg -t 1e-3 -x realization "$TEST_DIR/output.nc" \
```

Comment:
I've now corrected this test. The main point here is to compare the output between the case where percentiles are input directly to the ensemble calibration and where the percentiles have been rebadged as realizations and then input to the ensemble calibration. The data is expected to match for both outputs but the coordinate metadata will be different.

Reply:
@Katie-Howard @fionaRust I understand that this particular edit might interact with #881, as I've specified the name "percentile_over_realization", but we can talk about this.

Reply:
Where did we get to with this?  I don't want to make more work for this PR, but do we want some follow-on work to update the test inputs / output?

Reply:
I've amended this, now that #881 has been merged, as we should now specify `percentile` rather than `percentile_over_realization`, as the coordinate to the excluded from the comparison.